README.md

```diff
@@ -1,2 +1,39 @@
-# sass-bundle
-Delightful Sass Support for Symfony + AssetMapper
+# Sass For Symfony!
+
+This bundle make it easy to use Sass with Symfony's AssetMapper Component
+(no Node required!).
+
+- Automatically downloads the correct Sass binary
+- Adds a `sass:build` command to build and watch your Sass changes
+
+## Installation
+
+Install the bundle
+
+```shell
+composer require symfonycasts/sass-bundle
+```
+
+## Usage
+
+To use generated CSS, start by including the input file
+(`assets/styles/app.css` by default)
+
+```php
+{# templates/base.html.twig #}
+
+{% block stylesheets %}
+    <link rel="stylesheet" href="{{ asset('styles/app.scss') }}">
+{% endblock %}
+```
+
+Then you can create your SCSS file (`assets/styles/app.scss` by default)
+And start writing your style.
+
+Then run the command:
+
+```shell
+php bin/console sass:build --watch
+```
+
+And that's it!
```

Comment:
We should include a few more sections - like we do over here - https://github.com/SymfonyCasts/tailwind-bundle#how-does-it-work